README.md

```diff
@@ -1,4 +1,3 @@
-
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to add more information to your README file so your project is well documented